### ISSUE TITLE
Hide dates & comments on posts

### ISSUE THREAD

**Issue by curtistimson** [curtistimson/hugo-theme-massively#42]
Copied from a post on #25 from @baianomauricio:

------

I apologize for hijacking this thread with a different question but I thought to be better to use this one and not open a new thread with a simple <newbie> question. 

Is there a better way to remove the dates throughout the site then simply deleting the lines referring to the dates (lines 6-9) on partials/posts/features.html ?

Another question is how to disable comments but there are probably many posts on stack overflow about it. I wonder if there is a [params] markdown to disable dates and comments. Something similar to the attached screenshot.

<img width="710" alt="screen shot 2018-07-04 at 1 02 58 pm" src="https://user-images.githubusercontent.com/2117150/42288888-b105d2be-7f8a-11e8-94c3-43b7a5d720f8.png">

I can't finish this without thanking you for the great template you have put together and contribution to the community.

**Comment by curttimson** [comment]
## Hide Comments

To hide comments you can remove the Disqus shortname from the `config.toml`.

### Pre v4.0.0

Remove:

```
[params.settings]	
     disqus = "disqusShortname"
```

### v4.0.0+

Remove:

```
disqusShortname = "disqusShortname"
```

## Hide Post Dates

Other than removing the dates from the front matter there is not currently a way of hiding the post dates.

I think adding additional configuration to do this sounds like a good idea.

However I would prefer a `HideDatesOnPosts` config so that by default they are not displayed, and then this is an optional parameter.

**Comment by curttimson** [comment]
@baianomauricio Comments can now be hidden on specific posts since #47 was merged which has formed [Release 4.2.0](https://github.com/curttimson/hugo-theme-massively/releases/tag/4.2.0)
